nixos/modules/services/web-apps/mealie.nix

```diff
@@ -60,6 +69,12 @@ in
         '';
       };
     };
+
+    trustedProxies = lib.mkOption {
```

Comment:
This option seams quite too specific, and can be spared if we simply add in the documentation "if you use proxy, ...blablabla... use the option `--forwarded-allow-ips` ...blablabla..."

I'd be in favor of creating a NixOS Wiki page for Mealie, and adding this specific case to it.

Reply:
I'd like a wiki page, but I probably won't have enough time to write it right now. We can drop the option if you think it's too specific.

Reply:
Sound fair, done!